tembo-operator/src/stacks/types.rs

```diff
@@ -15,7 +15,7 @@ use utoipa::ToSchema;
 #[derive(Clone, Debug, Default, Serialize, Deserialize, JsonSchema, PartialEq, ToSchema)]
 pub struct Stack {
     pub name: String,
-    pub compute_templates: Option<Vec<ComputeTemplate>>,
+    pub compute_constraints: Option<ComputeConstraint>,
```

Comment:
This does not end up down in the CRD? So it does not have a backwards compatibility issue by dropping a field I think

Reply:
Correct, its not on the CRD. The other "Stack" struct IS on the CRD though.
 
https://github.com/tembo-io/tembo/blob/7469149b3d88d5a7e5eda54daff9e3ba614109f1/tembo-operator/src/apis/coredb_types.rs#L59

I'll create backlog ticket to move src/stacks/types::Stack to the `tembo-stacks` project